lib/spack/docs/packaging_guide.rst

```diff
@@ -3602,21 +3602,23 @@ In this case the package is passed as the second argument, and ``self`` is the b
 
 .. _multiple_build_systems:
 
-^^^^^^^^^^^^^^^^^^^^^^
+----------------------
 Multiple build systems
-^^^^^^^^^^^^^^^^^^^^^^
+----------------------
 
-There are cases where a software actively supports two build systems, or changes build systems
+There are cases where a package actively supports two build systems, or changes build systems
```

Comment:
(nitpick) The word "software" here was used on purpose, since "package" is also overloaded as a Spack concept. The idea is the "software" needs multiple build systems, the "package" can deal with it.

Reply:
I find it weird to talk about "a software". Let's just stick to package, it's clear enough. You create one package with two build systems in spack.